### DISCUSSION TITLE
How to export PDF

### DISCUSSION THREAD

**Discussion by LiuYX1998** [Open-Systems-Pharmacology/Forum#1163] [Questions & Problems]
![image](https://user-images.githubusercontent.com/97145041/157195807-2604b959-ce65-4a6a-a30d-8da165d09eb5.png)

![image](https://user-images.githubusercontent.com/97145041/157194083-b8452a52-671c-4b3a-bb48-050727a9f462.png)

when i complete my model，i want to export the PDF of the model，but it failed.
Could you help me to solve the issue？
Kind regards

**Comment by msevestre** [comment]
There seems to be an error with your MikText installation (Which is installed with the suite)
Did you use the setup to install PK-Sim?

**Comment by LiuYX1998** [comment]
Thank you for your reply. Should I download the PK-sim again？




At 2022-03-15 04:03:56, "Michael Sevestre" ***@***.***> wrote:

There seems to be an error with your MikText installation (Which is installed with the suite)
Did you use the setup to install PK-Sim?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by msevestre** [reply]
>Did you use the setup to install PK-Sim?

How did you install the suite? Using the setup or the portable version?

**Comment by LiuYX1998** [comment]
I do not know which version I have used.
The version I  have used was downloaded on the website 

https://www.open-systems-pharmacology.org/













At 2022-03-15 21:31:26, "Michael Sevestre" ***@***.***> wrote:

Did you use the setup to install PK-Sim?

How did you install the suite? Using the setup or the portable version?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by msevestre** [reply]
So this is probably using the setup. In that case, miktek is installed but is corrupted. There is no easy way to fix this unfortunately. 
We will discontinue PDF Export in the next version. My suggestion would be to export the graphs to images and the results to excel

**Comment by LiuYX1998** [comment]
thank you. But how to export the graphs?
looking forward to your reply.
















At 2022-03-15 21:55:32, "Michael Sevestre" ***@***.***> wrote:

So this is probably using the setup. In that case, miktek is installed but is corrupted. There is no easy way to fix this unfortunately.
We will discontinue PDF Export in the next version. My suggestion would be to export the graphs to images and the results to excel

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
